cfmljure.cfc

```diff
@@ -40,13 +42,12 @@ component {
                        "#cmd.cd# #project#" & nl &
                        "#lein# classpath" & nl );
             var classpath = "";
-            // TODO: not sure if this is ACF-compatible...
-            execute name="#cmd.run#" arguments="#cmd.arg#" variable="classpath" timeout="#timeout#";
+            cfexecute( name="#cmd.run#", arguments="#cmd.arg#", variable="classpath", timeout="#timeout#" );
             // could be multiple lines so clean it up:
             classpath = listLast( classpath, nl );
             classpath = replace( classpath, nl, "" );
             // turn the classpath into a URL list:
-            var classpathParts = listToArray( classpath, server.separator.path );
+            var classpathParts = listToArray( classpath, javaLangSystem.getProperty( "path.separator" ) );
```

Comment:
Since path.separator was only used once, I didn't assign it to a local var.  Let me know if you would rather I do.